mirgecom/viscous.py

```diff
@@ -275,3 +280,37 @@ def get_viscous_cfl(discr, eos, dt, cv):
         The CFL at each node.
     """
     return dt / get_viscous_timestep(discr, eos=eos, cv=cv)
+
+
+def get_local_max_species_diffusivity(transport, eos, cv):
+    """Return the maximum species diffusivity at every point.
+
+    Parameters
+    ----------
+    transport: mirgecom.transport.TransportModel
+        A model representing thermo-diffusive transport
+    eos: mirgecom.eos.GasEOS
+        An equation of state implementing the speed_of_sound method
+    cv: :class:`~mirgecom.fluid.ConservedVars`
+        Fluid solution
+    """
+    actx = cv.array_context
+
+    if(transport is None or transport._d_alpha.size == 0):
+        return 0 * cv.mass / cv.mass
+
+    species_diffusivity = transport.species_diffusivity(eos, cv)
+    stacked_diffusivity = cl.array.stack([x[0] for x in species_diffusivity])
```

Comment:
We probably want to avoid the openCL specificness here. Instead, I would just use `stack` from the array context like this:
```suggestion
    actx = cv.array_context
    species_diffusivity = transport.species_diffusivity(eos, cv)
    stacked_diffusivity = actx.np.stack([x[0] for x in species_diffusivity])
```

Reply:
Switched to the actx version. I didn't realize you could do that; thank you so much for the fix.